<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="D11-KamalKant.local" name="tests.CommonAPITests" tests="48" failures="10" timestamp="30 Sep 2020 19:00:18 GMT" time="35.365" errors="0">
  <testcase name="getCollectionWithLocation" time="0.799" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffCount" time="0.749" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffCount" time="0.839" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [10]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.CommonAPITests.getCityWithDiffCount(CommonAPITests.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffCount -->
  <testcase name="getCollectionWithInvalidCityId" time="0.730" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCollection(Common.java:60)
at tests.CommonAPITests.getCollectionWithInvalidCityId(CommonAPITests.java:181)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCollectionWithInvalidCityId -->
  <testcase name="getCityWithDiffParam" time="0.685" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.790" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithLocation" time="0.678" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithLocation" time="0.832" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithInvalidQParam" time="0.773" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCities(Common.java:48)
at tests.CommonAPITests.getCityWithInvalidQParam(CommonAPITests.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithInvalidQParam -->
  <testcase name="getCityWithIdAndName" time="0.675" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.658" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.CommonAPITests.getCityWithDiffParam(CommonAPITests.java:77)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffParam -->
  <testcase name="getCityWithLocation" time="0.688" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithInvalidQParam" time="0.918" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCities(Common.java:48)
at tests.CommonAPITests.getCityWithInvalidQParam(CommonAPITests.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithInvalidQParam -->
  <testcase name="getCityWithDiffParam" time="0.653" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidQParam" time="0.666" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.783" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidQParam" time="0.685" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithInvalidIdNameCombination" time="0.805" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.CommonAPITests.getCityWithInvalidIdNameCombination(CommonAPITests.java:134)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithInvalidIdNameCombination -->
  <testcase name="getCityWithIdOnly" time="0.788" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithName" time="0.781" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.717" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithIdOnly" time="0.716" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.698" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.690" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffCount" time="0.722" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [10]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.CommonAPITests.getCityWithDiffCount(CommonAPITests.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffCount -->
  <testcase name="getCityWithIdAndName" time="0.828" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.702" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.710" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithName" time="0.802" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithLocation" time="0.847" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithName" time="0.680" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithIdOnly" time="0.686" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.723" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.789" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.787" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithInvalidLocation" time="0.644" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCollection(Common.java:60)
at tests.CommonAPITests.getCollectionWithInvalidLocation(CommonAPITests.java:166)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCollectionWithInvalidLocation -->
  <testcase name="getCityWithDiffParam" time="0.677" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at tests.CommonAPITests.getCityWithDiffParam(CommonAPITests.java:77)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCityWithDiffParam -->
  <testcase name="getCityWithIdAndName" time="0.703" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithLocation" time="0.709" classname="tests.CommonAPITests"/>
  <testcase name="getCitiesWithInvalidLocation" time="0.655" classname="tests.CommonAPITests">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiCalls.Common.getCities(Common.java:48)
at tests.CommonAPITests.getCitiesWithInvalidLocation(CommonAPITests.java:174)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- getCitiesWithInvalidLocation -->
  <testcase name="getCityWithName" time="0.724" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.671" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.647" classname="tests.CommonAPITests"/>
  <testcase name="getCollectionWithCityId" time="0.810" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithIdAndName" time="0.754" classname="tests.CommonAPITests"/>
  <testcase name="getCategoriesTest" time="0.839" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithDiffParam" time="0.752" classname="tests.CommonAPITests"/>
  <testcase name="getCityWithIdOnly" time="0.708" classname="tests.CommonAPITests"/>
</testsuite> <!-- tests.CommonAPITests -->
